src/components/Avatar.js

```diff
@@ -1,12 +1,76 @@
 import * as React from "react";
-import {Avatar as MUIAvatar} from "@mui/material";
+import { Avatar as MUIAvatar, Button, ClickAwayListener, Divider, Box, Popper, Typography } from '@mui/material';
+import { useRef, useState } from 'react';
+import "./Avatar.css";
+export default function Avatar({imageLink, logout, email, name}) {
+  const anchorRef = useRef(null);
+  const [open, setOpen] = useState(false);
+
+  const handleToggle = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = (event) => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
```

Comment:
Can we avoid this by wrapping MUIAvatar as well in ClickAwayListener? If not, then it is fine